i18n-helpers/src/bin/mdbook-translate-helper.rs

```diff
@@ -0,0 +1,192 @@
+use std::env;
+use std::fs;
+use std::io;
+use std::io::BufRead;
+use std::io::Read;
+use std::io::Write;
+use::std::path::Path;
+use std::process::Command;
+use anyhow::Context;
+use anyhow::Error;
+use chrono;
+use walkdir::WalkDir;
+use zip::result::ZipError;
+use zip::write::SimpleFileOptions;
+use std::fs::File;
+
+fn build_translation(locale: String, dest_dir: String) -> Result<(), Error>{
+  if locale == "en" {
+    println!("::group::Building English course");
+  } else {
+    let locale_ref = &locale;
```

Comment:
We don't need a separate variable for this -- `&locale` is shorter and more clear to me :)